Doc/library/asyncio-policy.rst

```diff
@@ -98,6 +98,14 @@ asyncio ships with the following built-in policies:
    is configured to use the default policy automatically.
 
 
+.. class:: WindowsSelectorEventLoopPolicy
```

Comment:
Oh, one more change... 5 lines above this line:

```
    The default asyncio policy.  Uses :class:`SelectorEventLoop`
    on both Unix and Windows platforms.
```

please change it to

```
    The default asyncio policy.  Uses :class:`SelectorEventLoop`
    on Unix and :class:`ProactorEventLoop` on Windows.

    .. versionchanged:: 3.8
       ProactorEventLoop is now default on Windows.
```

Reply:
done